ember-cli-build.js

```diff
@@ -21,27 +23,52 @@ module.exports = function(defaults) {
 };
 
 
+/** Generate RTL CSS files along side regular CSS. */
+class BroccoliSassWithRtl extends BroccoliSass {
```

Comment:
Would it be better to make a standalone broccoli-cssjanus plugin (similar to how autoprefixer is implemented)?